### DISCUSSION TITLE
Only 1 backup per day

### DISCUSSION THREAD

**Discussion by VIMVa** [lavinir/hassio-onedrive-backup#135] [General]
Am I doing something wrong?
Since update to 2.2.2, only one backup runs every day while I set 'Backup interval (days)' to 0.5 and 'Allowed hours' to 2-3,14-15.
One daily backup is made just after 2am. None between 14 and 16. Problem did not occur with the earlier setting layout.
I do see that my entry of 0.5 for the interval is modified automatically to 0,5 when I save, exit and return to the settings Web interface.
Anyone?

**Comment by rorypjm** [comment]
I think:
There are less than 12 hours from 03:59 to 14:00 - And less than 12 hours from 15:59 to 02:00 ...
So when the next time slot comes around, it is not kicking off, because not enough time has elapsed, since the last backup.
Try remove the allowed hours to test my theory - give it 48 hours to run.

**Comment by VIMVa** [comment]
I just forgot because of illnesses to give feedback here: apologies.
It has been working as expected (2 backups/day) after removing and re-installing the add-on a couple of days after starting this thread.
